troposphere/constants.py

```diff
@@ -202,3 +202,32 @@
 CACHE_C1_XLARGE = 'cache.c1.xlarge'
 
 CACHE_T1_MICRO = 'cache.t1.micro'
+
+#
+# Parameter types
+#
+STRING = 'String'
```

Comment:
So I worry about generic constants like this - making this just STRING assumes that there'd be no use of STRING elsewhere, and could lead to confusion.  I almost wonder if we shouldn't turn constants into a module, and have things like parameter constants, etc.